src/utils/validators.ts

```diff
@@ -13,6 +13,15 @@ import type { Mutator } from 'final-form'
 const required = (value: any) =>
   value ? undefined : 'Make sure this field is filled correctly'
 
+const noSpecialCharacters = (value: string) => {
+  const specialCharacters = getSpecialCharacters(value)
+  return specialCharacters.length > 0
+    ? `Cannot contain special characters; ${specialCharacters
+        .map((c) => `'${c}'`)
+        .join(', ')}`
```

Comment:
```suggestion
    ? "Only letters and numbers are allowed."
```

Issue: not so sure about the error message here, it's a little confusing with the semi colon followed by a special characters in question..

Maybe we can use something simpler and easier for the user to understand at just a glance, eg: "Only letters and numbers are allowed." Taking this from Microsoft 365